travis/install_python_packages.py

```diff
@@ -5,7 +5,7 @@
 
 if (sys.version_info.major == 2):
    print("Installing Python 2.7 specific packages")
-   subprocess.call(["conda", "install", "pyqt=4", "scipy=0.16.1", "future", "pylint", "numpy", "matplotlib", "pandas", "pyyaml", "jinja2"])
+   subprocess.call(["conda", "install", "pyqt=4", "scipy=0.16.1", "future", "pylint", "numpy", "matplotlib", "pandas", "pyyaml", "jinja2", "segyio"])
```

Comment:
`segyio` is not part of conda - so this must be installed with `pip` - hopefully this will work:
```
pip install --user segyio
```

Reply:
Trying to install segyio from libres (travis.yml) instead.

Reply:
Why can't you do it from here? I think all dependency installations should be in one place